mmv1/products/redis/terraform.yaml

```diff
@@ -106,6 +114,8 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       secondaryIpRange: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
         diff_suppress_func: 'secondaryIpDiffSuppress'
+      customerManagedKey: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
```

Comment:
Question: Will API return value for this field if the use does not set this field explicitly?

Reply:
Synced offline, API won't return value if it is not provided by users. Removed this part